modules/ocf_puppet/manifests/puppetserver.pp

```diff
@@ -39,6 +39,18 @@
     notify               => Service['puppetserver'],
   }
 
+  # Allow kubernetes secrets to access kubernetes secrets
+  puppet_authorization::rule { 'kubernetes-secrets':
+    match_request_path   => '^/puppet/v3/file_(content|metadata)s?/kubernetes-secrets$',
+    match_request_type   => 'regex',
+    match_request_method => ['get', 'post'],
```

Comment:
post?

Reply:
I copied it from the docker secrets, should it be removed there too?

Reply:
masters probably don't need `post` to add secrets. we would do that manually